website/docs/docs/build/incremental-models.md

```diff
@@ -361,7 +361,7 @@ with large_source_table as (
 ```
 
 :::info
-- You don't need to explicitly alias any columns with either `DBT_INTERNAL_DEST` ("old" data) or `DBT_INTERNAL_SOURCE` ("new" data).
+- You need to explicitly alias any columns with either `DBT_INTERNAL_DEST` ("old" data) or `DBT_INTERNAL_SOURCE` ("new" data).
```

Comment:
I think it would make sense to make the target audience here clear (in not exactly these words): This is an intermediate/advanced feature, primarily intended for performance tuning. If you're using this feature, we take you to be someone who's inspected the compiled SQL produced by your incremental model, who understands your database's optimizer pretty well, and you know exactly the bit of SQL that needs adding in exactly this spot.